goyave_test.go

```diff
@@ -60,6 +60,13 @@ func (suite *GoyaveTestSuite) TestGetAddress() {
 	suite.Equal("https://test.system-glitch.me", getAddress("https"))
 
 	suite.Equal(getAddress("http"), BaseURL())
+
+	config.Set("server.domain", "")
+	config.Set("server.host", "0.0.0.0")
+	config.Set("server.port", 1235.0)
```

Comment:
Shouldn't we use integers instead of floats for port?

Reply:
It is converted to integer automatically because the type of this configuration entry is set to `reflect.Int`. This test was written before the v3 configuration revamp.

Reply:
Ok i've pushed a commit which change these values #142